parts/nav-bftop.php

```diff
@@ -11,8 +11,20 @@
 				<?php get_template_part( 'parts/nav', 'bfsubmenu' ); ?>
             </div>
         </div>
-        <div class="cell" style="width: 54px; height: 54px;"> 
-          User
+        <div class="cell" style="width: 54px; height: 54px;">
+        <a href="<?php bp_loggedin_user_link(); ?>" data-toggle="user-dropdown"><?php bp_loggedin_user_avatar( 'type=full' ); ?></a>
+        <div class="dropdown-pane" id="user-dropdown" data-dropdown data-hover="true" data-hover-pane="true" data-auto-focus="true">
+        <ul>
```

Comment:
Likewise here, this ul (and everything inside of it) should be indented from the div above it.